test/integration/clinical/clinical.spec.ts

```diff
@@ -17,35 +17,36 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+import { MongoDBContainer } from '@testcontainers/mongodb';
+import AdmZip from 'adm-zip';
 import chai from 'chai';
-// needed for typescript
 import 'chai-http';
+import deepEqualInAnyOrder from 'deep-equal-in-any-order';
+import _ from 'lodash';
 import 'mocha';
-import { GenericContainer } from 'testcontainers';
+import mongoose from 'mongoose';
+import { Network } from 'testcontainers';
 import app from '../../../src/app';
 import * as bootstrap from '../../../src/bootstrap';
+import { ClinicalEntitySchemaNames } from '../../../src/common-model/entities';
+import { getClinicalEntitiesFromDonorBySchemaName } from '../../../src/common-model/functions';
+import { TsvUtils, notEmpty } from '../../../src/utils';
+import { JWT_ABCDEF, JWT_CLINICALSVCADMIN, TEST_PUB_KEY } from '../test.jwt';
 import {
-	cleanCollection,
-	insertData,
 	assertDbCollectionEmpty,
+	cleanCollection,
 	emptyDonorDocument,
+	insertData,
 } from '../testutils';
-import { TEST_PUB_KEY, JWT_CLINICALSVCADMIN, JWT_ABCDEF } from '../test.jwt';
-import mongoose from 'mongoose';
-import AdmZip from 'adm-zip';
-import { ClinicalEntitySchemaNames } from '../../../src/common-model/entities';
-import { TsvUtils, notEmpty } from '../../../src/utils';
-import { getClinicalEntitiesFromDonorBySchemaName } from '../../../src/common-model/functions';
-import deepEqualInAnyOrder from 'deep-equal-in-any-order';
-import _ from 'lodash';
 
 chai.use(require('chai-http'));
 chai.use(deepEqualInAnyOrder);
 chai.should();
 
 describe('clinical Api', () => {
 	let mongoContainer: any;
-	let dburl = ``;
+	let testNetwork: any;
+	let dbUrl = ``;
```

Comment:
backticks aren't needed here
```suggestion
	let dbUrl = '';
```